tests/default.nix

```
@@ -11,11 +11,11 @@ let
   autoArgs = pkgs // {
     inherit
       helpers
-      lib
       pkgsUnfree
       self
       system
       ;
+    lib = helpers.extendedLib;
```

```suggestion
    lib = helpers.extendedLib;
    prevLib = lib;
```

? This can then be used in the maintainers test to access `prevLib.maintainers`.


Alt names:
- `prevLib`
- `oldLib`
- `nixpkgsLib`
- `upstreamLib`

---

I was just wondering if we should do something like that just to store it for usage when we need to use it.